e2e/pvc.go

```diff
@@ -284,21 +284,29 @@ func deletePVCAndValidatePV(c kubernetes.Interface, pvc *v1.PersistentVolumeClai
 			int(time.Since(start).Seconds()))
 		pvc, err = c.CoreV1().PersistentVolumeClaims(nameSpace).Get(context.TODO(), name, metav1.GetOptions{})
 		if err == nil {
+			e2elog.Logf("PVC %s (status: %s) has not been deleted yet, rechecking...", name, pvc.Status)
+
 			return false, nil
 		}
 		if isRetryableAPIError(err) {
+			e2elog.Logf("failed to verify deletion of PVC %s (status: %s): %v", name, pvc.Status, err)
```

Comment:
logging pvc name along with namespace name helps

Reply:
that is included in already existing logs at the start of this sub-function